### ISSUE TITLE
onImgLoad called before beforeImgLoad when using base64 image data

### ISSUE THREAD

**Issue by FabriceReynolds** [jerryshew/react-image-cropper#10]
The onImgLoad  event triggers before beforeImgLoad when the src is set to base64 data. The is probably due to the data already present in memory and firing asynchronously.

**Comment by du5rte** [comment]
Having this issue as well, I used this hack for the time being
```js
handleImageLoad() {
	setTimeout(() => {
		this.avatar.src = this.cropper.crop();
	}, 100);
}
```

**Comment by jerryshew** [comment]
@FabriceReynolds @du5rte yes, base64 data is already present but beforeImgLoad is fired when component mounted
